releasenotes/notes/project-dynamics-2f848a5f89655429.yaml

```diff
@@ -0,0 +1,51 @@
+features:
+  - |
+    Added the the projected Variational Quantum Dynamics (p-VQD) algorithm, see
```

Comment:
This replicates the current PVQD docstring - I guess thats fine - it might be nice though to at least alter it to point at the PVQD class so that when browsing the release note there is a link to PVQD in the API Ref